### DISCUSSION TITLE
p216 リスト19.21の`Repo UserRegister`の`UserRegister`の定義が本文に記載がない

### DISCUSSION THREAD

**Discussion by youta32449999** [budougumi0617/go_todo_app#55] [Ideas]
リスト19.21で下記のような定義を行なっておりますが、`UserRegister`の定義が本文に記載されておりません。
```go
type RegisterUser struct {
	DB   store.Execer
	Repo UserRegister
}
```

**Comment by budougumi0617** [comment]
正誤表に（書籍増刷時、コードを追加するような組版が大幅に変更になる修正は難しいため、）サンプルコードを参照してほしい追記文章を追加しました。

細部まで見ていただき、複数のご指摘ありがとうございました。たいへん助かります。

https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.7
